src/components/InternalDropdown/index.js

```diff
@@ -73,6 +73,7 @@ const InternalDropdown = forwardRef((props, reference) => {
             }
             return containerRef.current.focus();
         },
+        contains: element => containerRef.current.contains(element),
```

Comment:
Remember we need to be careful in what we expose here.
In this case why we need this?

Reply:
It is needed in useOutsideClick hook, otherwise throws the error 'ref.current.contains' is not a function,  it is used there to check if the target of the click event is a descendant of the element being rendered by the InternalOverlay component